app/api/router.py

```diff
@@ -2,11 +2,11 @@
 from fastapi.encoders import jsonable_encoder
 from fastapi.responses import JSONResponse
 from typing import Union
-from app.services.schemas import ToolRequest, ChatRequest, Message, ChatResponse, ToolResponse
-from app.utils.auth import key_check
-from app.services.logger import setup_logger
-from app.api.error_utilities import InputValidationError, ErrorResponse
-from app.api.tool_utilities import load_tool_metadata, execute_tool, finalize_inputs
+from services.schemas import ToolRequest, ChatRequest, Message, ChatResponse, ToolResponse
+from utils.auth import key_check
+from services.logger import setup_logger
+from api.error_utilities import InputValidationError, ErrorResponse
+from api.tool_utilities import load_tool_metadata, execute_tool, finalize_inputs
```

Comment:
Fetch these utility functions from the `app` folder.

Reply:
fixed it